src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -257,8 +257,9 @@ struct LottieGenericProperty : LottieProperty
     //Property has an either keyframes or single value.
     Array<LottieScalarFrame<T>>* frames = nullptr;
     T value;
```

Comment:
It might be able to reuse T value.

Reply:
no, value can be used like here:
```
              "o": {
                "a": 0,
                "k": -40,   // this is the value
                "ix": 3,
                "x": "var $bm_rt;\n$bm_rt = $bm_sum($bm_mul(content('Group 1').content('trim control').end, -1.8), value);"
              },
```
as we discussed, I'trry ti move this extra value to expressions

Reply:
@hermet can you please suggest the place where to move this value? I considered 'LottieExpression' but don't succeed. I tried not to template `LottieExpression` - do you think can I consider such an approach?

Reply:
@mgrudzinska It’s probably something like this:
```
struct LottieExpression
{
   ...
    union {
        RGB24 color;
        Point point;
        float number;
        uint8_t opacity;
    } output;
    ...
```

```
    T operator()(float frameNo, LottieExpressions* exps)
    {
        if (exps && exp) {
            T& out = reinterpret_cast<T&>(exp->output);
            frameNo = _loop(frames, frameNo, exp);
            if (exps->result<LottieGenericProperty<T>>(frameNo, out, exp)) return out;
        }
        return operator()(frameNo);
    }
```

Ultimately, I don’t think this caching method will serve as a fundamental solution. We need to evaluate the values of the expressions as they are executed in a chain.